runtime/etc/init/opa.conf

```diff
@@ -0,0 +1,27 @@
+description "Open Policy Agent"
+
+stop on shutdown
+
+respawn
+
+script
+      eval $(/opt/taupage/bin/parse-yaml.py /meta/taupage.yaml "config")
+      export AVAIL_ZONE="$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)"
+      export REGION="`echo \"$EC2_AVAIL_ZONE\" | sed -e 's:\([0-9][0-9]*\)[a-z]*\$:\\1:'`"
```

Comment:
Please use `$()` instead of backticks, it makes it a bit easier to read. In general, fixing [shellcheck](https://www.shellcheck.net)'s complaints is a good idea when writing shell scripts.

Reply:
changed